docs/pages/reference/helm-reference/teleport-cluster.mdx

```diff
@@ -103,8 +103,8 @@ This name will be shown to Teleport users connecting to the Kubernetes cluster.
 The `teleport-cluster` chart deploys two sets of pods, one for the Auth Service
 and another for the Proxy Service.
 
-`auth` contains values specific for Auth Service pods. You can use it to set
-specific values for auth pods, taking precedence over chart-scoped values.
+`auth` allows to set chart values only for the auth Kubernetes resources.
+This is merged with chart-scoped values and takes precedence in case of conflict.
```

Comment:
```suggestion
`auth` allows you to set chart values only for Kubernetes resources related to the Teleport Auth Service.
This is merged with chart-scoped values and takes precedence in case of conflict.
```

Suggested for grammar and consistent Teleport product terms in the docs.